src/Components/Browser.JS/Microsoft.AspNetCore.Components.Browser.JS.npmproj

```diff
@@ -7,23 +7,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <WebpackInputs Include="src\**\*.ts" />
```

Comment:
I considered adding build incrementalism to the root Npm.Common.targets and got something working fairly well. Might be something to consider to make inner loops with `dotnet build` be nice.